python/lsst/jointcal/jointcal.py

```diff
@@ -98,11 +110,9 @@ class JointcalTask(pipeBase.CmdLineTask):
     RunnerClass = JointcalRunner
     _DefaultName = "jointcal"
 
-    def __init__(self, schema, *args, **kwargs):
-        """
-        @param schema source catalog schema for all the catalogs we will work with.
-        """
+    def __init__(self, profile_jointcal=False, *args, **kwargs):
```

Comment:
I'm surprised this works. I would not expect to find `*args` after a keyword argument, especially one with a default value.

Do you need to accept `*args` at all? I suggest getting rid of it and alway using keyword arguments.